libs/src/sdk/api/users/UsersApi.ts

```diff
@@ -84,15 +83,12 @@ export class UsersApi extends GeneratedUsersApi {
     }
 
     // Write metadata to chain
-    const metadataCid = await generateMetadataCidV1(updatedMetadata)
```

Comment:
this ok with @theoilie ?

Reply:
is this because single txn signup creates it on discovery or something with partial update? might be out of the loop with recent discovery changes

Reply:
Yeah the cid is generated on discovery now (with the partial update change). Except for user creation, which isn't supported by the sdk anyway